apim-operator/pkg/apis/wso2/v1alpha1/zz_generated.openapi.go

```diff
@@ -327,20 +330,20 @@ func schema_pkg_apis_wso2_v1alpha1_RateLimitingSpec(ref common.ReferenceCallback
 					},
 					"bandwidth": {
 						SchemaProps: spec.SchemaProps{
-							Ref: ref("github.com/apim-crd/apim-operator/pkg/apis/wso2/v1alpha1.Bandwidth"),
+							Ref: ref("github.com/example-inc/k8s-apim-operator/apim-operator/pkg/apis/wso2/v1alpha1.Bandwidth"),
 						},
 					},
 					"conditions": {
 						SchemaProps: spec.SchemaProps{
-							Ref: ref("github.com/apim-crd/apim-operator/pkg/apis/wso2/v1alpha1.Conditions"),
+							Ref: ref("github.com/example-inc/k8s-apim-operator/apim-operator/pkg/apis/wso2/v1alpha1.Conditions"),
 						},
 					},
 				},
-				Required: []string{"type", "timeUnit", "unitTime", "requestCount"},
+				Required: []string{"type", "timeUnit", "unitTime", "requestCount", "stopOnQuotaReach", "description", "bandwidth", "conditions"},
```

Comment:
Here we need only the earlier 4 fields as required. Other fields are optional in ratelimiting kind